packages/material-ui-lab/src/PaginationItem/PaginationItem.test.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { createMount, getClasses } from '@material-ui/core/test-utils';
+import describeConformance from '@material-ui/core/test-utils/describeConformance';
+import { createClientRender } from 'test/utils/createClientRender';
+import PaginationItem from './PaginationItem';
+
+describe('<PaginationItem />', () => {
```

Comment:
These all look great. I think the usage of `byTestId` is perfectly reasonable here.